features/step_definitions/description_change_steps.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "faker"
+
+Given("a new planning application with a description of {string}") do |string|
```

Comment:
Can you try and make these override `@planning_application` instead? The idea is that during the span of an example (scenario, spec, whatever you want to call it), we refer to the same instance variable throughout our steps, it's effectively part of our state.

Something akin to:

```
Given a new planning application # step already exists
And the planning application has a description of "Foobar"
```